### PR TITLE
Update rate limit for faucet

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Updated rate-limit for STX faucet.

### CODE REVIEW

File: content/feature-guides/rate-limiting.md [hunk 1]
```diff
@@ -19,4 +19,4 @@ You can refer to the rate limit for each endpoint in the table below:
 
 ## STX faucet
 
-The Stacks faucet rate limits depend on the type of request. For stacking requests, there is a limit of **1 request per 2 days**. In case of regular Stacks faucet requests, the limits are set to **1 request per minute**.
+The Stacks faucet rate limits depend on the type of request. For stacking requests, there is a limit of **1 request per 2 days**. In case of regular Stacks faucet requests, the limits are set to **1 request per day**.
```
Comment (CharlieC3) [inline content/feature-guides/rate-limiting.md:22]: I don't think we actually have a different rate limit depending on whether it's a stacking request or not. It's always 1 request per day.

**Comment by andresgalante** [comment]
We've increased the rate limit of the faucet for the hackathon. My questions are:

- Did we bring it back down again?
- if so, what's the rate limit so I can update this docs and merge it?

**Comment by CharlieC3** [comment]
@andresgalante We did not yet bring the rate limit down. We were going to have @aravindgee handle that change as part of [this PR](https://github.com/hirosystems/k8s/pull/449#issuecomment-1787590984), but we can implement that sooner if you'd like.

**Comment by zone117x** [comment]
Seems like this isn't needed anymore.

**Pull request closed without merging**